firestore/query.go

```diff
@@ -569,8 +579,55 @@ func trunc32(i int) int32 {
 	return int32(i)
 }
 
+// Get returns an array of query's resulting documents.
+func (q Query) Get(ctx context.Context) ([]*DocumentSnapshot, error) {
+	limitedToLast := false
+
+	if q.limitToLast != nil {
+		// Flip Query order statements before posting a request.
+		limitedToLast = true
+		q.limit = q.limitToLast
+		q.limitToLast = nil
+
+		for i := range q.orders {
+			if q.orders[i].dir == Asc {
+				q.orders[i].dir = Desc
+			} else {
+				q.orders[i].dir = Asc
+			}
+		}
+		// Swap cursors.
+		q.startVals, q.endVals = q.endVals, q.startVals
+		q.startDoc, q.endDoc = q.endDoc, q.startDoc
+		q.startBefore, q.endBefore = q.endBefore, q.startBefore
+	}
+	docIterator := &DocumentIterator{
+		iter: newQueryDocumentIterator(withResourceHeader(ctx, q.c.path()), &q, nil),
+	}
```

Comment:
Why not to call method `Documents()`straight to return `DocumentIterator`?

Reply:
Done